src/components/forms/DevconGrantsForm.tsx

```diff
@@ -0,0 +1,1015 @@
+              maxLength: 32768
+            })}
+          />
+
+          {errors?.projectDescription?.type === 'maxLength' && (
+            <Box mt={1}>
+              <PageText as='small' fontSize='helpText' color='red.500'>
+                Project description cannot exceed 32768 characters.
+              </PageText>
+            </Box>
+          )}
+        </FormControl>
+
+        <Flex direction={{ base: 'column', lg: 'row' }}>
+          <Controller
+            name='eventType'
+            control={control}
+            defaultValue={{ value: '', label: '' }}
+            render={({ field: { onChange } }) => (
+              <FormControl id='event-type-control' isRequired mb={8} mr={{ md: 12 }}>
+                <FormLabel htmlFor='eventType'>
+                  <PageText display='inline' fontSize='input'>
+                    What type of event is this?
+                  </PageText>
+                </FormLabel>
+
+                <Select
+                  id='eventType'
+                  options={EVENT_TYPE_OPTIONS}
+                  onChange={onChange}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </FormControl>
+            )}
+          />
+
+          <Controller
+            name='eventFormat'
+            control={control}
+            rules={{
+              required: true,
+              validate: selected => selected.value !== ''
+            }}
+            defaultValue={{ value: '', label: '' }}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <FormControl id='event-format-control' isRequired mb={8}>
+                <FormLabel htmlFor='eventFormat'>
+                  <PageText display='inline' fontSize='input'>
+                    Is your event in-person or online?
+                  </PageText>
+                </FormLabel>
+
+                <Select
+                  id='eventFormat'
+                  options={EVENT_FORMAT_OPTIONS}
+                  onChange={value => {
+                    onChange(value);
+                    setEventFormat(value);
+                  }}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+
+                {error && (
+                  <Box mt={1}>
+                    <PageText as='small' fontSize='helpText' color='red.500'>
+                      Event format is required.
+                    </PageText>
+                  </Box>
+                )}
+              </FormControl>
+            )}
+          />
+        </Flex>
+
+        <Box display={isInPerson ? 'block' : 'none'}>
```

Comment:
Maybe hybrid event type should trigger this conditional field too, as it includes in person

Reply:
Uhmm good point. Will do.